iron-request.html

```diff
@@ -440,6 +440,23 @@
     _updateStatus: function() {
       this._setStatus(this.xhr.status);
       this._setStatusText((this.xhr.statusText === undefined) ? '' : this.xhr.statusText);
+    },
+
+    /**
+     * Returns the given response header in way that won’t cause an error during
+     * CORS requests if it is not returned or accessible due to
+     * `Access-Control-Expose-Headers`.
+     *
+     * @param {string} header The HTTP response header to return.
+     * @return {string|null} The header value or null if it is not present.
+     */
+    _responseHeader: function(header) {
+      var allHeaders = this.xhr.getAllResponseHeaders();
+      if (!!allHeaders &&
+          allHeaders.toLowerCase().indexOf(header.toLowerCase()) !== -1) {
```

Comment:
There are a couple things that are not great about this solution, but it’s the simplest.
+ Linear time lookup on the string returned by `getAllResponseHeaders` on every request.
+ It will trigger the logic to grab the response (and error out) if it is a CORS request, the _value_ of any header is the same as the key being looked for, and `Access-Control-Expose-Headers` doesn’t contain it.